lib/resources/dstu2/goal.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Goal
+field_name_base_url: http://hl7.org/fhir/DSTU2/goal-definitions.html#Goal
+fields:
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: Codes for grouping and sorting goals.
```

Comment:
This should be "Indicates a category the goal falls within." per http://hl7.org/fhir/goal-definitions.html#Goal.category